src/model/Epic.java

```diff
@@ -35,29 +36,25 @@ public void calculateStatus() {
         boolean isNew = false;
         boolean isInProgress = false;
         boolean isDone = false;
-        if (this.getAllSubTasks() != null){
-            for(Subtask subtask : this.getAllSubTasks()){
-                if (subtask.getStatus() == Progress.IN_PROGRESS){
+
+        if (this.getAllSubTasks() != null) {
+            for (Subtask subtask : this.getAllSubTasks()) {
+                if (subtask.getStatus() == Progress.IN_PROGRESS) {
                     isInProgress = true;
-                }
-                else if (subtask.getStatus() == Progress.DONE){
+                } else if (subtask.getStatus() == Progress.DONE) {
                         isDone = true;
-                }
-                else {
+                } else {
                     isNew = true;
                 }
             }
-            if (isNew == true && isDone == false && isInProgress == false ){
+            if (isNew == true && isDone == false && isInProgress == false) {
```

Comment:
Поля `isNew`, `isDone` и `isInProgress` имеют тип boolean, поэтому дополнительно сравнивать его с значениями `true` и `false` не нужно
```suggestion
            if (isNew && !isDone && !isInProgress) {
```